utilities/sanity.py

```
@@ -315,6 +316,15 @@ def cluster_sanity(
             hco_namespace=hco_namespace,
         )
 
+        if request.session.config.getoption(skip_hco_taint_check):
+            LOGGER.warning(f"Skipping HCO taint check, got {skip_hco_taint_check}")
+        else:
+            LOGGER.info(
+                f"Check HCO TaintedConfiguration. (To skip HCO taint check pass {skip_hco_taint_check} to pytest)"
+            )
+            if taint_conditions := is_hco_tainted(admin_client=admin_client, hco_namespace=hco_namespace.name):
+                raise ClusterSanityError(err_str=f"HCO has TaintedConfiguration: {taint_conditions}.")
```

Ruff (0.16.1)

[warning] Abstract `raise` to an inner function

(TRY301)